#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
uint16 byte_count
uint16 packet_count
uint16 bad_bytes
uint16 old_packets